### DISCUSSION TITLE
PCB and plates thickness and material questions.

### DISCUSSION THREAD

**Discussion by tekkshaman** [strayer/taira-keyboard#12] [General]
Hello,
Thanks for sharing this awesome project.
I'm currently in process of ordering pcbs and plates from jlcpcb to build taira keyboard. I've stumbled upon several very basic questions I couldn't find answers for:
- Is default 1.6mm a good thickness for both pcbs and top/bottom plates? 
- Could I order top and bottom plates in aluminum? Is it a good idea in the first place?
- BOM mentions Spacer M2 4mm, but according parts link is pointing at 2x3mm spacer, am i missing something? 

Thanks again.

**Comment by strayer** [comment]
Hey, I converted the issue to a discussion, hope you don't mind.

> Is default 1.6mm a good thickness for both pcbs and top/bottom plates?

Yep!

> Could I order top and bottom plates in aluminum? Is it a good idea in the first place?

I'm not sure, to be honest. I guess aluminum could be electrically conductive? At least the top plate could theoretically short contact pads because of the flippable PCB design. You could cover the inner side with something like kapton tape, I guess…

> BOM mentions Spacer M2 4mm, but according parts link is pointing at 2x3mm spacer, am i missing something?

The article on AliExpress has multiple variants, one of them being m2x4. I checked my order history to be sure and I ordered m2x4 as well.

By the way, I have to give a heads up that I'm currently working on a Rev2.0 that will include a possibly bigger battery (200 mAh) and nice!view displays on both halves. I'm not sure when I'll be done with that, its at least a few weeks or even months based on the little free time I have, but it will at least require a new PCB.

If you go forward with building the Rev1.0, feel free to let me know if you need any help!

**Reply by tekkshaman** [reply]
Thank you very much for your answer, and sorry for mislabeling. I'll probably proceed with v1 and it seems like i've got the rest of it more or less figured out. Rev 2 does sound interesting, but i'm not sure if i have enough patience to wait. Btw, are there any plans to add any kind of pointing device to v2 or later since ZMK rolling out their new input subsystem?

**Reply by strayer** [reply]
Again, apologies for the _very_ late answer 🙈 No, I don't plan to add any pointing devices. I'm very spoiled by big trackballs and the Apple Magic Trackpad, so I personally wouldn't benefit from them, even though I find them intriguing.
